crates/tinymist/src/actor/export.rs

```diff
@@ -78,7 +80,12 @@ impl ExportActor {
             'accumulate: loop {
                 log::debug!("RenderActor: received request: {req:?}");
                 match req {
-                    ExportRequest::ChangeConfig(cfg) => self.config = cfg,
+                    ExportRequest::ChangeConfig(config) => {
```

Comment:
I think we can remove the entry field from `ExportConfig` for more clear code, if whatever we start to tend to update config separately from entry here.